### ISSUE TITLE
Detect panics within code base

### ISSUE THREAD

**Issue by gcmurphy** [HewlettPackard/gas#7]
Could be useful when auditing a code base to know about cases which call into panic without a recover clause.

**Comment by gcmurphy** [comment]
I guess the thinking here was that there are perhaps some cases where a panic would have security implications such as Denial of Service. It could be worth keeping this for specific things like HTTP request / network handlers. @ccojocar What do you think? Keep or ditch?

**Comment by ccojocar** [comment]
I agree that it could be useful to detect the panics especially for DoS. Any thoughts how we can do this reliably?

**Comment by jpgoldberg** [comment]
Should this also detect `log.Fatalf()`?

**Comment by ccojocar** [comment]
@gcmurphy Do you think can we detect the panics by just blacklisting the functions/methods which possibly panic inside? In addition, we need to check that there isn't any recovery clause.
